Mage.Sets/src/mage/cards/s/ShivanDevastator.java

```diff
@@ -0,0 +1,47 @@
+package mage.cards.s;
+
+import mage.MageInt;
+import mage.abilities.common.EntersBattlefieldAbility;
+import mage.abilities.effects.common.EntersBattlefieldWithXCountersEffect;
+import mage.abilities.keyword.FlyingAbility;
+import mage.abilities.keyword.HasteAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.counters.CounterType;
+
+import java.util.UUID;
+
+/**
+ * @author TheElk801
+ */
+public final class ShivanDevastator extends CardImpl {
+
+    public ShivanDevastator(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{X}{R}");
+
+        this.subtype.add(SubType.DRAGON);
+        this.subtype.add(SubType.HYDRA);
+        this.power = new MageInt(0);
+        this.toughness = new MageInt(0);
+
+        // Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        // Haste
+        this.addAbility(HasteAbility.getInstance());
```

Comment:
append with comma?

Reply:
was this a new feature that was added while I was gone?

Reply:
Sorry I was thinking of `.concatBy()`, but that's only for effects.